shared/src/endpoints.ts

```diff
@@ -23,6 +23,19 @@ export enum Endpoints {
   deleteComment = 'deleteComment',
 }
 
+export function WithParams(endpoint: EndpointConfig, ...params: string[]): EndpointConfig {
+  let url = endpoint.url
+  const placeholders = url.split("/").filter(token => token.startsWith(":"))
```

Comment:
a regex would be a little cleaner here I think, something like: `placeholders = url.match(/:[^\/]*/g)`